src/lib/Interfaces.ts

```diff
@@ -27,6 +27,9 @@ export interface IGoogleOAuth2Credential {
  */
 export interface IIdentity {
   uniqueID?: string | undefined;
+  displayName?: string | undefined;
+  verified?: boolean | undefined;
```

Comment:
TLDR from my long response below is s/verified/verifiedIdentity (to match what's in VDR), and remove `financialAccounts` (not needed in this schema yet). Disregard note about moving to IUser. Hope this helps!

Reply:
@un7c0rn +1 and done